### PR TITLE
Fix typo in Kotlin doc for contract method

### PR DESCRIPTION
**Pull request by xaviarias**
Fix the typo in "Specifies the contact of a function."

### CODE REVIEW

File: libraries/stdlib/src/kotlin/contracts/ContractBuilder.kt [hunk 1]
```diff
@@ -146,4 +146,4 @@ public enum class InvocationKind {
 @InlineOnly
 @SinceKotlin("1.3")
 @Suppress("UNUSED_PARAMETER")
-public inline fun contract(builder: ContractBuilder.() -> Unit) { }
```
Comment (ilya-g) [inline libraries/stdlib/src/kotlin/contracts/ContractBuilder.kt]: Please revert unrelated line ending change

Comment (xaviarias) [reply libraries/stdlib/src/kotlin/contracts/ContractBuilder.kt]: OK done

Comment (ilya-g) [reply libraries/stdlib/src/kotlin/contracts/ContractBuilder.kt]: Hmm, strange, the new line is still there.
Ok, I'll merge the commit manually and clear it up.

**Comment by ilya-g** [comment]
Thank you for spotting that. Merged manually.
